src/main/java/gov/acwi/wqp/etl/citMeth/CitMethProcessor.java

```diff
@@ -0,0 +1,28 @@
+package main.java.gov.acwi.wqp.etl.citMeth;
+
+import org.springframework.batch.item.ItemProcessor;
+
+public class CitMethProcessor implements ItemProcessor<CitMeth, CitMeth> {
+
+	final static String TRIM = "^\\s]*|\\s]*$";
+	
+	private String trimString(String value) {
```

Comment:
I'm fairly certain that there is be a better way to handle this. There  are values in the result set that are null. I used this function to prevent Null Pointer exceptions and to appropriately return null for a null value.